src/target/riscv/encoding.h

```diff
@@ -33,15 +32,18 @@
 #define SSTATUS_UPIE        0x00000010
 #define SSTATUS_SPIE        0x00000020
 #define SSTATUS_SPP         0x00000100
+#define SSTATUS_VS          0x00000600
```

Comment:
So the vector context status position within MSTATUS is based on the latest draft of the vector spec, and not the 1.11 privileged spec?  Makes sense that the vector is more up to date on that detail, but just wanted to double check.

Reply:
Yes, and this matches Spike.